### DISCUSSION TITLE
Want to contribute?

### DISCUSSION THREAD

**Discussion by Akascape** [Akascape/TkDial#1] [Announcements]
Do you want to contribute? 

- Fix the progress arc problem in imageknob

- ~~Making a Image Knob (where we can add our own images as the knob):~~ [IMPLEMENTED]

![](https://static-s.aa-cdn.net/img/gp/20600012825583/yvQgMpnLIf1MjB9KkwxzDl8kmbs7_tZvH8CESeTf9yXqPEwrl4gPs0S6gnmf_m3k6w=s300?v=1)

- ~~Making **knob/Jogwheel** widget~~ [IMPLEMENTED]

![example](https://user-images.githubusercontent.com/89206401/202974696-73987406-c4b9-4a72-8bc5-caa1525099aa.png)

- ~~Making a **meter** widget~~ [IMPLEMENTED]

![meter](https://user-images.githubusercontent.com/89206401/202974754-dd3cd4ca-79ed-40a6-a347-61cf7505f7eb.png)

-  ~~Making a circular **progress bar**~~ [IMPLEMENTED]

![progressbar](https://i.stack.imgur.com/LLAJ1.png)

I will try to make this in future if I get the time. Otherwise, you can try yourself!

**Comment by adamigom** [comment]
Hey I am very keen on knob/Jogwheel only this item is missing for my project will you be creating one in the near future?

**Reply by Akascape** [reply]
@adamigom Not working on it currently, planning to work on mid-2023 maybe.

**Reply by adamigom** [reply]
This is not good information for me , remains to wait thanks for the information.

**Reply by Akascape** [reply]
Hello @adamigom , knob/jogwheel is finally implemented. please update the package.

**Reply by adamigom** [reply]
Many thanks I'm getting on with finishing the project.
